google/cloud/bigtable/doc/environment-variables.dox

```diff
@@ -0,0 +1,20 @@
+/*!
+@page bigtable-environment-variables Environment Variables Used by the Google Cloud Bigtable C++ Client.
+
+The Cloud Bigtable C++ Client library introduces a single environment variable
+that affects its behavior. If `BIGTABLE_EMULATOR_HOST` is set, it specifies the
+endpoint (typically a `host:port` string) for the [Cloud Bigtable
+Emulator](https://cloud.google.com/bigtable/docs/emulator). And configure the
```

Comment:
s/Emulator/emulator/ (it's not a product or a proper name — it's capitalized on the page only in the heading)

Reply:
Done.